pkg/volume/csi/csi_attacher.go

```diff
@@ -51,6 +51,10 @@ type csiAttacher struct {
 	csiClient csiClient
 }
 
+type waitForStatus func(attach *storage.VolumeAttachment, volumeHandle string) (bool, error)
+
+type waitForStatusInternal func(volumeHandle, attachID string, timer *time.Timer, timeout time.Duration) (string, error)
```

Comment:
This type it not used (well, it is used, but parameter of this type is not used).